### PR TITLE
Initial design contacts

### PR DESCRIPTION
**Pull request by clenrock**
Basics of the app. There is a main screen with a list of dummy names and a button to make a Bluetooth connection.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ericcornelissen) [inline .idea/misc.xml]: This file is being ignored by the `.gitignore`, please remove it manually 🙂

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ericcornelissen) [inline .idea/vcs.xml]: This file is being ignored by the `.gitignore`, please remove it manually 🙂

**Comment by jverbraeken** [comment]
Merge conflicts 🎉

**Comment by ericcornelissen** [comment]
@clenrock, to get these logs locally you should install Gradle ([details here](https://gradle.org/install#manually)) and run `gradle build connectedCheck` from your the root of the repository. It will generate the report for you and put it in `.../NervousFish/app/build/reports/lint-results.html` 😉

**Comment by clenrock** [comment]
@ericcornelissen Thanks for the tip, I'll fix my errors :)

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ericcornelissen) [inline app/src/main/res/layout/activity_main.xml]: Why is this changed to a `<merge>` ?

File: app/src/main/res/menu/menu_main.xml [hunk 4]
```diff
@@ -1,10 +0,0 @@
-<menu xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ericcornelissen) [inline app/src/main/res/menu/menu_main.xml]: Do we need the file if it is empty?

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (clenrock) [inline app/src/main/res/layout/activity_main.xml]: The report told me that the frame layout was unnecessary at that location. I also thought it was a bit weird to do at the activity_main, for content_main it is understandable.

File: app/src/main/res/menu/menu_main.xml (hunk 4 above)
Comment (clenrock) [inline app/src/main/res/menu/menu_main.xml]: I thought I removed this menu, I'll check again

Comment (clenrock) [inline app/src/main/res/menu/menu_main.xml]: This file is removed already. It's strange how GitHub shows it. It looks like it's an empty file, but it's totally gone

Comment (ericcornelissen) [inline app/src/main/res/menu/menu_main.xml]: 👍

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (ericcornelissen) [inline app/src/main/res/layout/activity_main.xml]: What report?

Comment (clenrock) [inline app/src/main/res/layout/activity_main.xml]: The gradle build lint report
